Zabolotskiy/hash.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Regarding all binary files above - they are not really needed :).
Please add them to .gitignore next time.